docs/consuming-any-data.adoc

```diff
@@ -1,6 +1,6 @@
 Consuming Arbitrary Topic
 =========================
-:base_version: 0.0.42
+:base_version: 0.0.43
```

Comment:
As a process, the person who to make a release will "fix" this version before releasing so please change this to `0.0.43-SNAPSHOT` instead. (overlooked last time)